components/inputs/input-checkbox.js

```diff
@@ -1,16 +1,58 @@
 import '../colors/colors.js';
 import { css, html, LitElement } from 'lit-element/lit-element.js';
-import { checkboxStyles } from './input-checkbox-styles.js';
```

Comment:
I pulled these styles into here as part of this change as they were only referenced by the list component. This will cut down on an extra web request in most scenarios.